backend/code_review_backend/issues/models.py

```diff
@@ -11,7 +11,12 @@
 
 LEVEL_WARNING = "warning"
 LEVEL_ERROR = "error"
-ISSUE_LEVELS = ((LEVEL_WARNING, "Warning"), (LEVEL_ERROR, "Error"))
+LEVEL_CRITICAL = "critical"
+ISSUE_LEVELS = (
+    (LEVEL_WARNING, "Warning"),
+    (LEVEL_ERROR, "Error"),
+    (LEVEL_CRITICAL, "Critical"),
+)
```

Comment:
What if, for simplicity, we keep only two levels: `error` and `warning`? `error` for CI breaking issues, `warning` for all other issues which developers can feel free to ignore.

Reply:
And we'd temporarily force mozlint issues to be errors and all others to be warnings, until we fix it in mozilla-central

Reply:
They already are errors ! that's why i added that extra level

Reply:
Yep, but the others can be errors too at the moment, instead we could make them warnings and consider "error" as breaking CI and "warning" as not breaking CI. This way we reduce the changes we need to make (both here and in mozilla-central), without losing any feature (as we are currently not using the level basically).

Reply:
Ok, works for me

Reply:
Done, i simply removed the backend patch & the critical level (in 5fdfac8)